mobile/src/main/java/ch/epfl/sdp/mobile/application/chess/ChessFacade.kt

```diff
@@ -199,7 +208,28 @@ private data class StoreMatch(
       }
 
   override suspend fun update(game: Game) {
-    store.collection("games").document(id).update { this["moves"] = game.toAlgebraicNotation() }
+    val chess = store.collection("games").document(id).get<ChessDocument>()
```

Comment:
I'm not sure `chess` is the most suitable name for this. What about `document` for instance ?

Reply:
is done